sdk/cosmosdb/cosmos/test/public/integration/split.spec.ts

```diff
@@ -122,7 +122,7 @@ describe("Partition Splits", () => {
         .container(container.id)
         .items.query("SELECT * FROM root r", {
           maxItemCount: 2,
-          maxDegreeOfParallelism: 1,
+          maxDegreeOfParallelism: 2,
```

Comment:
Why we changed maxDegreeOfParallelism to 2?

Reply:
so earlier before when maxDegreeOfParallelism was 1, the header value use to be true, which would internaly use parallelExecutionQueryContext. Then when I introduced this check that if maxDegreeOfParallelism is 1 , then the header should be false, the test started failing because default query execution context gets called. so changed maxDegreeOfParallelism to 2.